### PR TITLE
Refactor to make the project more compliant with the code style guide

### PR DESCRIPTION
**Pull request by stevk**
Required changes for #475

### CODE REVIEW

File: Source/Manifest.cs [hunk 1]
```diff
@@ -9,17 +9,17 @@ internal class Manifest
         public string Folder;
         public List<Model> Models = new List<Model>();
 
-        // Model group, to be listed in the manifest as the folder name
+        // Model group, to be listed in the manifest as the folder name.
```
Comment (bghgary) [inline Source/Manifest.cs]: nit: i would only put periods for sentences

File: Source/ModelGroup.cs [hunk 2]
```diff
@@ -154,17 +164,17 @@ public static List<Runtime.MeshPrimitive> CreateMultiPrimitivePlane(bool include
...
+                            new Vector3( 0.5f,  0.5f, 0.0f)
                         },
                         TextureCoordSets = includeTextureCoords ? new List<List<Vector2>>
                         {
                             new List<Vector2>
                             {
-                                new Vector2( 0.0f, 1.0f),
-                                new Vector2( 1.0f, 1.0f),
-                                new Vector2( 1.0f, 0.0f)
+                                new Vector2(0.0f,  1.0f),
```
Comment (bghgary) [inline Source/ModelGroup.cs]: Revert this as there is no negative sign

File: Source/ModelGroup_MultiNode.cs [hunk 3]
```diff
@@ -5,486 +5,530 @@ namespace AssetGenerator
...
+                            new Runtime.MeshPrimitive
+                            {
+                                Positions = vertexPositions,
+                                TextureCoordSets = textureCoordSets,
+                                Indices = indices0,
+                            }
+                        }
+                    },
+                    Name = "Node0",
+                    Children = new []
```
Comment (bghgary) [inline Source/ModelGroup_MultiNode.cs]:
```suggestion
                    Children = new[] 
```

File: Source/ModelGroup_SkinE.cs [hunk 4]
```diff
@@ -25,26 +25,26 @@ public static List<Runtime.Node> CreatePlaneWithSkinE(bool jointsHaveCommonParen
                             {
                                 Positions = new List<Vector3>()
                                 {
-                                    // Trunk
-                                    new Vector3(-0.125f, 0.0f,-0.25f),
-                                    new Vector3( 0.125f, 0.0f,-0.25f),
-                                    new Vector3(-0.125f, 0.0f, 0.00f),
-                                    new Vector3( 0.125f, 0.0f, 0.00f),
+                                    // Trunk.
+                                    new Vector3(-0.125f, 0.0f, -0.25f),
```
Comment (bghgary) [inline Source/ModelGroup_SkinE.cs]: nit: use 3 decimals for all numbers here

File: Source/ModelGroups/Animation_NodeMisc.cs [hunk 5]
```diff
@@ -298,15 +298,15 @@ void CreateMultipleChannelsForDifferentNodes(List<Runtime.AnimationChannel> chan
                 }),
                 CreateModel((properties, channels, nodes, animations) => {
                     // Rotate the model, and then apply the same target animation to it (Animation overrides)
-                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3, 0, 0);
+                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3, 0.0f, 0.0f);
```
Comment (bghgary) [inline Source/ModelGroups/Animation_NodeMisc.cs]:
```suggestion
                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3.0f, 0.0f, 0.0f);
```

File: Source/ModelGroups/Animation_NodeMisc.cs [hunk 6]
```diff
@@ -298,15 +298,15 @@ void CreateMultipleChannelsForDifferentNodes(List<Runtime.AnimationChannel> chan
...
                     SetRotationChannelTarget(channels[0], nodes[0]);
                     SetLinearSamplerForConstantRotation(channels[0]);
                     properties.Add(new Property(PropertyName.Description,
                         "There is one channel that targets a node. The node has a rotation of <code>[0.0,&nbsp;0.5,&nbsp;0.0,&nbsp;0.866]</code>. The channel overrides the rotation of the node to a different constant value of <code>[0.0,&nbsp;-0.5,&nbsp;0.0,&nbsp;0.866]</code>."));
                 }),
                 CreateModel((properties, channels, nodes, animations) => {
                     // Rotate the model, and then apply an translation animation to it (Animation doesn't override rotation)
-                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3, 0, 0);
+                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3, 0.0f, 0.0f);
```
Comment (bghgary) [inline Source/ModelGroups/Animation_NodeMisc.cs]:
```suggestion
                    nodes[0].Rotation = Quaternion.CreateFromYawPitchRoll(FloatMath.Pi / 3.0f, 0.0f, 0.0f);
```

File: Source/ModelGroups/Animation_NodeMisc.cs [hunk 7]
```diff
@@ -315,17 +315,18 @@ void CreateMultipleChannelsForDifferentNodes(List<Runtime.AnimationChannel> chan
...
+                    var channel = new Runtime.AnimationChannel();
                     animations.Add(new Runtime.Animation
                     {
                         Channels = new[]
                         {
-                            new Runtime.AnimationChannel()
+                            channel
                         }
                     });
                     SetRotationChannelTarget(channels[0], nodes[0]);
```
Comment (bghgary) [inline Source/ModelGroups/Animation_NodeMisc.cs]: Move the next two lines up, since the rest of it is a contiguous group

File: Source/Program.cs [hunk 8]
```diff
@@ -103,24 +102,24 @@ private static void Main(string[] args)
                 readme.WriteOut(executingAssembly, modelGroup, modelGroupFolder);
                 manifestMaster.Add(manifest);
 
-                // Write out the manifest JSON specific to this model group.
+                // Writes out the manifest JSON specific to this model group.
```
Comment (bghgary) [inline Source/Program.cs]: nit: I like the comment how it was. This is inconsistent throughout the code.

Comment (stevk) [reply Source/Program.cs]: Reverted. This class should be consistent with itself now at least.

File: Source/Program.cs [hunk 9]
```diff
@@ -55,42 +57,41 @@ private static void Main(string[] args)
...
+                for (var comboIndex = 0; comboIndex < numCombos; comboIndex++)
                 {
                     var model = modelGroup.Models[comboIndex];
                     var filename = $"{modelGroup.Id}_{comboIndex:00}.gltf";
 
                     using (var data = new Data($"{modelGroup.Id}_{comboIndex:00}.bin"))
                     {
-                        // Passes the desired properties to the runtime layer, which then coverts that data into
-                        // a gltf loader object, ready to create the model
+                        // Passe the desired properties to the runtime layer, which then coverts that data into
```
Comment (bghgary) [inline Source/Program.cs]:
```suggestion
                        // Pass the desired properties to the runtime layer, which then coverts that data into
```

**Pull request merged**